pkgs/development/tools/yourkit-java/default.nix

```
@@ -0,0 +1,83 @@
+{ fetchzip
+, lib
+, stdenv
+, copyDesktopItems
+, imagemagick
+, makeDesktopItem
+, jre
+}:
+let
+  vPath = v: lib.elemAt (lib.splitString "-" v) 0;
+
+  arch = let inherit (stdenv.targetPlatform) system;
+         in if system == "x86_64-linux" then "x64"
+            else if system == "aarch64=linux" then "arm64"
+            else throw "Unsupported system";
+
+  desktopItem = makeDesktopItem {
+    name = "Yourkit JavaProfiler";
+    exec = "yourkit-java-profiler";
+    comment = "Award winning, fully featured low overhead profiler for Java EE and Java SE platforms";
+    desktopName = "YourKit JavaProfiler";
+    type = "Application";
+    icon = "yourkit-java-profiler";
+    terminal = false;
+    categories = [ "Development" ];
+    startupWMClass = "yourkit-java-profiler";
```

```suggestion
    startupWMClass = "YourKit Java Profiler";
```

As above. With this maybe `startupNotify = false` can be removed?